test/test_context.h

```diff
@@ -15,12 +15,14 @@ namespace ArduinoContextTest
 
     void test_reset(void)
     {
+#if false // Unit test is unreliable: assumes memory is not recycled.
```

Comment:
Can you please clarify why this is necessary ?

Why only failing now with the move to `platformio test` ?

Reply:
> Why only failing now with the move to `platformio test`?

Please confirm the tests are currently run - looking at the action log (E.g. https://github.com/FabioBatSilva/ArduinoFake/actions/runs/17988942467/job/51174123656), I don't see any test output so I'm not sure they are currently running.

Reply:
Yes, should be running here : https://github.com/FabioBatSilva/ArduinoFake/actions/runs/17988942467/job/51174123656#step:3:124

Reply:
After enabling [verbose test output](https://github.com/adbancroft/ArduinoFake/tree/test_tests), I can see the current test *[does fail](https://github.com/adbancroft/ArduinoFake/actions/runs/18040282943/job/51337111336#step:3:186)*, even though CTest [reports a pass](https://github.com/adbancroft/ArduinoFake/actions/runs/18040282943/job/51337111336#step:3:188).   :worried: